course_content/iris_course/6.Data_Processing.ipynb

```diff
@@ -215,91 +436,131 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "1\\. Load as cubes the datasets found at `iris.sample_data_path('E1_north_america.nc')` and `iris.sample_data_path('A1B_north_america.nc')`. Print the summary of each cube."
+    "#### 1\\. Load data\n",
+    "Load as cubes the datasets found at `iris.sample_data_path('E1_north_america.nc')` and `iris.sample_data_path('A1B_north_america.nc')`. Print the summary of each cube."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
-   "source": []
+   "source": [
+    "# user code ..."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# SAMPLE SOLUTION\n",
+    "# %load solutions/iris_exercise_6.3a"
+   ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "2a\\. Plot the following data in a single figure with three maps in one row:\n",
+    "----\n",
+    "\n",
+    "#### 2a\\. Plot E1, A1B and difference data\n",
     "\n",
-    " * the air temperature in the E1 scenario in 2099, \n",
-    " * the air temperature in the A1B scenario in 2099, and\n",
+    "Plot the following data in a single figure with three maps, side-by-side in one row :\n",
+    "\n",
+    " * the air temperature in the E1 scenario for the year 2099, \n",
+    " * the air temperature in the A1B scenario for the year 2099, and\n",
     " * the difference between the two scenarios.\n",
     "\n",
-    "Think about the most appropriate matplotlib colormap(s) to use for each plot. Hint: the different matplotlib colormaps can be seen at https://matplotlib.org/1.5.3/examples/color/colormaps_reference.html. "
+    "Think about the most appropriate matplotlib colormap(s) to use for each plot.\n",
+    "\n",
+    "Hint: the different matplotlib colormaps can be seen at https://matplotlib.org/1.5.3/examples/color/colormaps_reference.html. \n",
+    "\n",
+    "2b\\. What information do your plots show? "
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
-   "source": []
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
    "source": [
-    "2b\\. What information do your plots show? "
+    "# user code ..."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
-   "source": []
+   "source": [
+    "# SAMPLE SOLUTION\n",
+    "# %load solutions/iris_exercise_6.3b"
+   ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "3\\. Produce cubes that describe the area-averaged air temperature over time for each scenario. Calculate the model difference between these two cubes."
+    "----\n",
+    "\n",
+    "#### 3. Produce time sequences of global area-averaged air temperature\n",
```

Comment:
I don't like the area-averaging question here. The trouble is it is introducing a new concept in the exercises, which I would prefer if it only tested what the person would have already learnt.

Reply:
:+1:  good points !!